apps/minifront/src/components/staking/account/delegation-value-view/staking-actions/index.test.tsx

```diff
@@ -26,17 +26,15 @@ const zeroBalance = new ValueView({
 const validatorInfo = new ValidatorInfo({ validator: {} });
 
 let MOCK_STAKING_TOKENS_AND_FILTER: {
-  unstakedTokensByAccount: Map<number, ValueView | undefined>;
+  stakingTokens: ValueView | undefined;
```

Comment:
`useStakingTokensAndFilter()` now returns an object with `stakingTokens` and `accountSwitcherFilter` properties, whereas before the properties were `unstakedTokensByAccount` and `accountSwitcherFilter`. Aside from the rename of `unstakedTokens` to `stakingTokens`, that property is also now a single `ValueView` representing the staking tokens for a given account (which is passed into the `useStakingTokensAndFilter()` hook as an argument), rather than being a map of account numbers to staking token `ValueView`s